content/java/concepts/array-list/terms/size/size.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: '.size()'
+Description: 'Returns the number of elements in the ArrayList.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `size()` method of the `ArrayList` class returns the number of elements in the list.
+
+## Syntax
+
+```pseudo
+int size() = list.size()
+```
+
+## Example
+
```

Comment:
We'd like to see a sentence describing what the example does to introduce the example.

Reply:
alright I will add now :)